wsd/COOLWSD.cpp

```diff
@@ -2353,6 +2361,30 @@ void COOLWSD::innerInitialize(Application& self)
     // Allow UT to manipulate before using configuration values.
     UnitWSD::get().configure(conf);
 
+    // net::Defaults
+    {
+        typedef int int_t;
```

Comment:
Seem pedantic. If we must, we can use `int64_t` and be done. This covers potentially large values. Other types are virtually impossible to be used with integral config values coming as strings.

Reply:
a left over .. b/c int64_t didn't work